daily-code-update/array.c

```diff
@@ -1,21 +1,87 @@
-#include<stdio.h>
+#include <stdio.h>
+#include<stdlib.h>
 #include<conio.h>
 
-int main( ) 
-{ 
-    int avg, sum = 0 ; 
-    int i ; 
-    int marks[6] ; /* array declaration */ 
-    
-    for ( i = 0 ; i <= 5 ; i++ ) 
-    { 
-    printf ( "\nEnter marks " ) ; 
-    scanf ( "%d", &marks[i] ) ; /* store data in array */ 
-    } 
-    
-    for ( i = 0 ; i <= 5 ; i++ ) 
-    sum = sum + marks[i] ; /* read data from an array*/ 
-    avg = sum / 6 ; 
+// Function prototypes
```

Comment:
why are you updating same code again.. I think you have written this average code earlier and now you have overridden the same file..

Reply:
Your PR title doesn't make sense.
You can add description in PR where you add screenshot. Because title should be small (72 char) but additional details can be added in PR description

Reply:
I have to read code to understand what you are doing. rather your PR title and description should be clear and complete for faster review

Reply:
> Your PR title doesn't make sense. You can add description in PR where you add screenshot. Because title should be small (72 char) but additional details can be added in PR description

okay
i have made the changes

Reply:
> I have to read code to understand what you are doing. rather your PR title and description should be clear and complete for faster review

changes made